src/site/content/en/blog/native-file-system/index.md

```diff
@@ -216,27 +281,28 @@ takes a string, which is what we want for a text editor. But it can also take a
 [BufferSource][buffersource], or a [Blob][blob]. Finally, finish writing by
 calling `FileSystemWriter.close()`.
 
-{% Aside 'caution' %}
-  There's no guarantee that the contents are written to disk until
-  the `close()` method is called.
-{% endAside %}
+{% endDetails %}
 
-### What else is possible?
+### Storing file handles in IndexedDB
 
-Beyond reading and writing files, the Native File System API provides
-several other new capabilities.
+Starting in Chrome 82, you can save a file handle to IndexedDB. This makes
```

Comment:
Either write out Indexed database or enclose it in backticks.

Reply:
Can't we just called it IndexedDB? It's not actually the API interface, but the API name.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API):
> IndexedDB is a low-level API for client-side storage of significant amounts of structured data, including files/blobs.